src/riscv/lib/src/pvm/tezos.rs

```diff
@@ -20,6 +23,14 @@ use tezos_smart_rollup_constants::riscv::SBI_TEZOS_INBOX_NEXT;
 use tezos_smart_rollup_constants::riscv::SBI_TEZOS_REVEAL;
 use tezos_smart_rollup_constants::riscv::SbiError;
 
+// temporary till added kernel_sdk constants
+
+/// Function ID for `sbi_tezos_secp256k1_verify`
+pub const SBI_TEZOS_SECP256K1_VERIFY: u64 = 0x0a;
+
+/// Function ID for `sbi_tezos_keccak_hash256`
```

Comment:
The function is called Keccak-256 so I think the names/comments should reflect that here (e.g., `SBI_TEZOS_KECCAK256_HASH`) and in the rest of the code.